packages/abi-gen-wrappers/package.json

```diff
@@ -13,10 +13,11 @@
         "build": "yarn pre_build && tsc -b",
         "build:ci": "yarn build",
         "lint": "tslint --format stylish --project .",
+        "artifacts_transform": "node ../monorepo-scripts/lib/contract_artifacts_transform.js ../contract-artifacts/artifacts && prettier --write ../contract-artifacts/artifacts/*.json",
```

Comment:
Let's remove this from here and also move this script into the `contract-artifacts` package.